src/debug/debug.proto

```diff
@@ -98,6 +98,10 @@ message System {
   repeated App loki_logs = 6;
   repeated App binaries = 7;
   repeated App profiles = 8;
+  message Defaults {
```

Comment:
I think this should go in the DumpV2Request message instead since it's an Application concept. The System message was meant to contain non-pachyderm concepts like the DB state, logging, etc...

Reply:
Ah, that wasn’t clear. Updated!